modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/import/CellPreview.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import React from 'react';
+
+const CellPreview = ({cell, cellIndex,}) => {
+
+	return (
+		<ClayTable.Cell key={cellIndex}>
```

Comment:
There's no need for a key here. 

Please have a looks at the [documentation](https://reactjs.org/docs/lists-and-keys.html)